Index: asymptote-2.53/settings.cc
===================================================================
--- asymptote-2.53.orig/settings.cc	2019-08-21 06:51:14.000000000 +0200
+++ asymptote-2.53/settings.cc	2019-08-21 23:14:28.830527539 +0200
@@ -96,7 +96,7 @@
 string defaultPDFViewer="open";
 #else  
 string defaultPSViewer="gv";
-string defaultPDFViewer="acroread";
+string defaultPDFViewer="xdg-open";
 #endif  
 string defaultGhostscript="gs";
 string defaultGhostscriptLibrary="";
