---
 settings.cc |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- asymptote.orig/settings.cc
+++ asymptote/settings.cc
@@ -100,9 +100,9 @@ string defaultPSViewer="open";
 string defaultPDFViewer="open";
 string defaultHTMLViewer="open";
 #else
-string defaultPSViewer="gv";
-string defaultPDFViewer="acroread";
-string defaultHTMLViewer="google-chrome";
+string defaultPSViewer="xdg-open";
+string defaultPDFViewer="xdg-open";
+string defaultHTMLViewer="xdg-open";
 #endif
 string defaultGhostscript="gs";
 string defaultGhostscriptLibrary="";
