Description: Hard code PATH_MAX on Gnu Hurd
Origin: John Bowman <bowman@ualberta.ca>
Forwarded: https://github.com/vectorgraphics/asymptote/commit/48f313effd296591c3c44155736ba98f03a68535
Author: John Bowman <bowman@ualberta.ca>
Last-Update: 20220108

diff --git a/lspserv.cc b/lspserv.cc
index 85ea986a..f544261d 100644
--- a/lspserv.cc
+++ b/lspserv.cc
@@ -87,6 +87,9 @@ std::string wslUnix2Dos(std::string const& unixPath)
     bool isMntPath=false;
     char drive;
 
+#ifdef __GNU__
+#define PATH_MAX 4096
+#endif
     char actPath[PATH_MAX];
     (void) realpath(unixPath.c_str(), actPath);
     std::string fullPath(actPath);
