--- asymptote.orig/picture.cc
+++ asymptote/picture.cc
@@ -596,7 +596,7 @@
   cmd.push_back("-dBATCH");
   cmd.push_back("-P");
   if(safe)
-    cmd.push_back("-dSAFER");
+    cmd.push_back("-dSAFER -dDELAYSAFER"); // Support transparency extensions.
   cmd.push_back("-sDEVICE=pdfwrite");
   cmd.push_back("-dEPSCrop");
   cmd.push_back("-dSubsetFonts=true");
@@ -612,6 +612,11 @@
   cmd.push_back("-dDEVICEHEIGHTPOINTS="+String(max(b.top-b.bottom,3.0)));
   push_split(cmd,getSetting<string>("gsOptions"));
   cmd.push_back("-sOutputFile="+stripDir(pdfname));
+  if(safe) {
+    cmd.push_back("-c");
+    cmd.push_back(".setsafe");
+    cmd.push_back("-f");
+  }
   cmd.push_back(stripDir(epsname));
 
   char *oldPath=NULL;
