--- asymptote-3.01.orig/Makefile.in
+++ asymptote-3.01/Makefile.in
@@ -166,7 +166,7 @@
 	echo @set Datadir @datadir@ >> $@
 	echo @set Docdir @docdir@ >> $@
 
-asymptote.so: $(COREFILES:=.pic.o) glew.o
+asymptote.so: $(COREFILES:=.pic.o) -lGLEW
 	$(CXX) $(OPTS) -shared -o asymptote.so revision.o $(COREFILES:=.pic.o) $(SHAREDLIBS)
 
 $(LSP_BUILD_ROOT)/liblspcpp.a:
--- asymptote-3.01.orig/configure.ac
+++ asymptote-3.01/configure.ac
@@ -647,8 +647,8 @@
    AC_DEFINE(HAVE_LIBGL,1,[libgl is enabled])
    AC_DEFINE(FREEGLUT,1,[Freeglut is enabled])
    AC_DEFINE(HAVE_LIBGLUT,1,[Freeglut library is available])
-   INCL=$INCL" -Ibackports/glew/include"
-   GLEW="glew.o "
+   # INCL=$INCL" -Ibackports/glew/include"
+   GLEW="-lGLEW "
    ADD_VCPKG_LIB_FOR_PKGCONFIG(glut)
 else  # managed by the system
    AC_CHECK_HEADERS([ncurses/curses.h ncurses.h curses.h],[break])
@@ -662,8 +662,8 @@
               AC_CHECK_HEADER(GLUT/glut.h, [AC_DEFINE(HAVE_LIBGLUT,1,
                    DEFINE_LIB[GLUT])
                            LIBS=$LIBS"-framework GLUT -framework OpenGL -framework Cocoa "
-                           INCL=$INCL" -Ibackports/glew/include"
-                           GLEW="glew.o "],
+                           # INCL=$INCL" -Ibackports/glew/include"
+                           GLEW="-lGLEW "],
                                  AC_MSG_NOTICE([*** Could not find GLUT: will compile without OpenGLLUT support ***]))
         ;;
      *)
@@ -673,8 +673,8 @@
                                  [AC_DEFINE(HAVE_LIBGL,1,
                              DEFINE_LIB([GL]))
                              LIBS=$LIBS"-lGL "
-                             GLEW="glew.o "
-                             INCL=$INCL" -Ibackports/glew/include"],
+                             GLEW="-lGLEW "
+                             # INCL=$INCL" -Ibackports/glew/include"],
                                    AC_MSG_NOTICE([*** Could not find libGL: will compile without OpenGL support ***]))
 esac
    if test "x$enable_offscreen" = "xyes"; then
