---
 Makefile.in  |    2 +-
 configure.ac |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- asymptote.orig/Makefile.in
+++ asymptote/Makefile.in
@@ -115,7 +115,7 @@ endif
 
 export prefix docdir exampledir mandir infodir INSTALL MAKE DESTDIR TEXI2DVI
 
-asy:	version glew.o
+asy:	version
 	if test -n "$(MSDOS)"; then \
           $(CXX) $(OPTS) -o $(NAME) $(FILES:=.o) revision.o asy.o $(DOSLIBS); \
 	else \
--- asymptote.orig/configure.ac
+++ asymptote/configure.ac
@@ -414,7 +414,7 @@ case "$OSTYPE" in
          AC_CHECK_LIB([GL], [glDepthMask],
                                  [AC_DEFINE(HAVE_LIBGL,1,
                              DEFINE_LIB([GL]))
-                             LIBS=$LIBS"-lGL glew.o "],
+                             LIBS=$LIBS"-lGL -lGLEW "],
                                    AC_MSG_NOTICE([*** Could not find libGL: will compile without OpenGL support ***]))
 esac
    if test "x$enable_offscreen" = "xyes"; then
