Description: We link dynamically w/ libtinyexr.
Author: Hilmar Preusse <hille42@web.de>
Forwarded: not-needed
Last-Update: 2025-03-02

--- asymptote.orig/configure.ac
+++ asymptote/configure.ac
@@ -628,9 +628,9 @@
    )
 fi
 
-TINYEXR_ROOT=tinyexr
-INCL=$INCL" -I$TINYEXR_ROOT"
-AC_SUBST(TINYEXR_ROOT)
+#TINYEXR_ROOT=tinyexr
+#INCL=$INCL" -I$TINYEXR_ROOT"
+#AC_SUBST(TINYEXR_ROOT)
 
 if test "x$enable_gl" != "xno"; then
 if test "x$with_vcpkg" != "xno"; then  # managed by vcpkg
